### PR TITLE
Create weekly-update.2015-05-01.md

### PR DESCRIPTION
**Pull request by Gioyik**
CC #65

### CODE REVIEW

**Comment by Gioyik** [comment]
There are not **Known issues** and **Notable changes** in this release due v2.0 changelog is not release. If you think we should add something there, let me know.

**Comment by Gioyik** [comment]
We have a lot of community updates :)

File: weekly-updates/weekly-update.2015-05-01.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+* initial [repo](https://github.com/jasnell/node.js-convergence) for code convergence work for io.js/node.js.
+* Node.js Foundation Governance/Convergence Call on [Youtube](https://www.youtube.com/watch?v=u9h0s3YtNAU).
+* Microsoft releases new [Visual Studio Code](https://code.visualstudio.com/) based on Electron (AtomShell).
+* io.js support coming to [Modulus](http://blog.modulus.io/upcoming-updates-for-nodejs-applications)
+
+### Upcoming Events
+
+* [NodeConf Adventure](http://nodeconf.com/) tickets are on sale, June 11th - 14th at Walker Creek Ranch, CA
+* [CascadiaJS](http://2015.cascadiajs.com/) tickets are on sale, July 8th - 10th at Washington State
+* [BrazilJS Conf]() tickets are on sale, August 21st - 22nd at Shopping Center BarraShoppingSul
```
Comment (yosuke-furukawa) [inline weekly-updates/weekly-update.2015-05-01.md]: Add url http://braziljs.com.br/

**Comment by yosuke-furukawa** [comment]
please add my `iojs-new-feature` link if you recognize it worthy ... >_<

https://github.com/iojs/evangelism/issues/65#issuecomment-97400601
https://github.com/yosuke-furukawa/iojs-new-features

**Comment by Gioyik** [comment]
Sure, I recognize it. I was waiting to know if someone says to add it as Notable Changes, but I will add it as Community Update and wait for official Changelog.

File: weekly-updates/weekly-update.2015-05-01.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+* io.js v2.0 release candidate is [out](https://iojs.org/download/nightly/v2.0.0-nightly20150501b4ad5d7050/).
+* io.js [ARM cluster](https://twitter.com/rvagg/status/593226114992087041) is growing.
+* Onboarding another batch of committers [iojs/io.js#1552](https://github.com/iojs/io.js/issues/1552)
+* io.js TC meeting 4/29 on [Youtube](https://www.youtube.com/watch?v=-e675TT4WEA) and [Soundcloud](https://twitter.com/dotproto/status/594145574204510208).
+* io.js/node.js code [convergence plan draft](https://github.com/jasnell/dev-policy/blob/master/convergence.md).
+* initial [repo](https://github.com/jasnell/node.js-convergence) for code convergence work for io.js/node.js.
+* Node.js Foundation Governance/Convergence Call on [Youtube](https://www.youtube.com/watch?v=u9h0s3YtNAU).
+* Microsoft releases new [Visual Studio Code](https://code.visualstudio.com/) based on Electron (AtomShell).
+* io.js support coming to [Modulus](http://blog.modulus.io/upcoming-updates-for-nodejs-applications)
+* Yosuke Furukawa created [iojs-new-features repo](https://github.com/yosuke-furukawa/iojs-new-features) with upcomming changes in io.js v2.0.
```
Comment (watilde) [inline weekly-updates/weekly-update.2015-05-01.md]: Maybe minor typo in this desc:

``` diff
- upcomming
+ upcoming
```

File: weekly-updates/weekly-update.2015-05-01.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+### Community Updates
+
+* New [johnny-five](http://johnny-five.io/) site launched.
+* iojs.io domain was donated to the project and it will redirect to iojs.org [iojs/io.js#1534](https://github.com/iojs/io.js/issues/1534)
+* david-dm.org is [now running](https://twitter.com/_alanshaw/status/592855646124531713) on iojs
+* io.js v2.0 release candidate is [out](https://iojs.org/download/nightly/v2.0.0-nightly20150501b4ad5d7050/).
+* io.js [ARM cluster](https://twitter.com/rvagg/status/593226114992087041) is growing.
+* Onboarding another batch of committers [iojs/io.js#1552](https://github.com/iojs/io.js/issues/1552)
+* io.js TC meeting 4/29 on [Youtube](https://www.youtube.com/watch?v=-e675TT4WEA) and [Soundcloud](https://twitter.com/dotproto/status/594145574204510208).
```
Comment (diagramatics) [inline weekly-updates/weekly-update.2015-05-01.md]: Nitpicking on the capitalization. The Tube in YouTube and Cloud in SoundCloud.

File: weekly-updates/weekly-update.2015-05-01.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+* io.js/node.js code [convergence plan draft](https://github.com/jasnell/dev-policy/blob/master/convergence.md).
+* initial [repo](https://github.com/jasnell/node.js-convergence) for code convergence work for io.js/node.js.
+* Node.js Foundation Governance/Convergence Call on [Youtube](https://www.youtube.com/watch?v=u9h0s3YtNAU).
```
Comment (diagramatics) [inline weekly-updates/weekly-update.2015-05-01.md]: The Tube in this YouTube also.

**Comment by Gioyik** [comment]
typo fixes done

**Comment by Gioyik** [comment]
Seems everything is fine. @julianduque could you check this and take action of publish on Medium? Thanks

**Comment by yosuke-furukawa** [comment]
LGTM

**Comment by julianduque** [comment]
Published: https://medium.com/node-js-javascript/io-js-week-of-may-1st-362e01678d48

**Pull request merged**